flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -1369,18 +1256,8 @@ public void testMaximumRegistrationDurationAfterConnectionLoss() throws Exceptio
 		configuration.setString(TaskManagerOptions.REGISTRATION_TIMEOUT, "100 ms");
 		final TaskSlotTable taskSlotTable = new TaskSlotTable(Collections.singleton(ResourceProfile.UNKNOWN), timerService);
 
-		final long heartbeatInterval = 10L;
 		final TaskManagerServices taskManagerServices = new TaskManagerServicesBuilder().setTaskSlotTable(taskSlotTable).build();
-		final TaskExecutor taskExecutor = new TaskExecutor(
-			rpc,
-			TaskManagerConfiguration.fromConfiguration(configuration),
-			haServices,
-			taskManagerServices,
-			new HeartbeatServices(heartbeatInterval, 10L),
-			UnregisteredMetricGroups.createUnregisteredTaskManagerMetricGroup(),
-			null,
-			dummyBlobCacheService,
-			testingFatalErrorHandler);
+		final TaskExecutor taskExecutor = createTaskExecutor(taskManagerServices);
```

Comment:
here we had `new HeartbeatServices(10L, 10L)`, not `HEARTBEAT_SERVICES `, should we keep it?

Reply:
Yes, we should make it shorter time in this case, so pass custom heartbeat service here.